autogpt/exceptions/exception_handler.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from autogpt.exceptions import HandledException
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    """A custom exception handler that manages HandledException"""
+    if issubclass(exc_type, HandledException):
+        sys.stderr.write(f"{exc_value.message}\n")
+        sys.exit(1)
```

Comment:
Why would it exit on a handled exception?
I think the original issue was about leaving it up to the user to continue?

Reply:
True, removed.

Reply:
sys.exit(1) is still there?

Reply:
Uhm...no? I don't see it. The snapshot says it is "outdated".